pkg/uploader/kopia/progress.go

```diff
@@ -177,3 +177,7 @@ func (p *Progress) EstimationParameters() upload.EstimationParameters {
 func (p *Progress) Enabled() bool {
 	return true
 }
+
+func (p *Progress) GetIncrementalSize() int64 {
```

Comment:
I suggest we move this function outside and keep this `Progress` pure.
Anyway, this incremental size is not directly reported to `Progress` and is out of the logic of it.

Reply:
It is also fine. I just suggested we get the `estimatedBytes` and `cachedBytes` from `Progress` and calculate `incrementalBytes` outside. But as long as we call `GetIncrementalSize` in place, it also looks fine.